Portland 2024 Schedule (#2098)
writethedocs/www: docs/conf/portland/2024/news/announcing-schedule.rst

:template: {{year}}/generic.html

> .. post:: Feb 26, 2024

Reviewer: ```suggestion
.. post:: Feb 29, 2024
```

Leap day!

   :tags: {{year}}, {{shortcode}}-{{year}}, schedule

Announcing Full Schedule
========================

Hey folks!

The {{city}} conference is less than two months away, and we're busy making things happen behind the scenes.

> You've already seen our list of :doc:`awesome speakers </conf/portland/{{year}}/speakers/>`, now we can also show you the schedule so you know when they will be presenting!

Reviewer: ```suggestion
You've already seen our list of :doc:`awesome speakers </conf/portland/{{year}}/speakers/>`, now we're announcing our full schedule so you know when they will be presenting!
```

Full presentation schedule
--------------------------

The conference website is now updated with the :doc:`schedule </conf/portland/{{year}}/schedule/>` for the main stage presentations, so you can start planning your time (and getting excited!) for the conference. 

We're keeping the moderated Q&A session after each speaker that we introduced last year. You'll have a chance to ask our speakers questions after the talk via our wonderful set of moderators.

Now is a great time to buy your `tickets <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/tickets/>`_ if you haven't already.
Also please read over our `Health and Safety policy <https://www.writethedocs.org/conf/portland/{{year}}/health/>`__,

> so you understand the measures we're taking to keep our attendees safe.

Reviewer: Not 100% sure we want to highlight this here, since we aren't doing much this year? 🤷

Author: yeah, already nixed

Join us for our hike again this year
------------------------------------

> We're reviving the traditional Pittock Mansion `hike <https://www.writethedocs.org/conf/portland/{{year}}/outing/>`_ on Saturday. This is a great way to experience the beauty that Portland holds within city limits, with a walk along a creek, a majestic view of Mount Hood, and finishing with a meader through Hoyt Arboritum with hundreds of different trees and plants. 

Reviewer: ```suggestion
We're doing our traditional Pittock Mansion `hike <https://www.writethedocs.org/conf/portland/{{year}}/outing/>`_ on Saturday. This is a great way to experience the beauty that Portland holds within city limits, with a walk along a creek, a majestic view of Mount Hood, and finishing with a meader through Hoyt Arboritum with hundreds of different trees and plants. 
```

Not sure reviving is the right framing?

All our events and activities
-----------------------------

Write the Docs is more than a conference. Every year we make sure to have a variety of activities that our attendees can enjoy, whether you want to learn something, meet new people, or share knowledge.

The main events that you can participate in Monday & Tuesday are:

* **Lightning Talks.** We'll have `lightning talks <https://www.writethedocs.org/conf/portland/{{year}}/lightning-talks/>`__ each conference day after lunch. These are 5 minute presentations and a lot of fun. They can be about anything you want! This is your chance to share something you are passionate about with the audience. Slides are not required.
* **Unconference.** We'll be running the `Unconference <https://www.writethedocs.org/conf/portland/{{year}}/unconference/>`_ as our formal "second track" at the conference. This is a place to schedule discussions about a specific topic, so you can find other attendees who care about the same niche topics that you love. Want to chat with like-minded folks about oxford commas or a specific docs as code tool? This is the place.

> * **Social event**. On Monday night we'll be hosting a quiet social event. This is a great chance to get to know more folks in the community and have a drink or two on us. Non-alcoholic options will be very available, as well as some snacks.

Reviewer: ```suggestion
* **Social event**. On Monday night we'll be hosting a social event. This is a great chance to get to know more folks in the community and have a drink or two on us. Non-alcoholic options will be very available, as well as some snacks.
```

Not sure quiet is the vibe we're going for :)
